charts/cinder-csi-plugin/templates/controllerplugin-statefulset.yaml

```diff
@@ -106,7 +106,7 @@ spec:
             - name: CSI_ENDPOINT
               value: unix://csi/csi.sock
             - name: CLOUD_CONFIG
-              value: /etc/kubernetes/cloud-config
+              value: /etc/config/cloud.conf
```

Comment:
This doesnt work , if secret is added via hostpath, L138. May be we have to remove the option to be added by hostpath

Reply:
Fixed in https://github.com/kubernetes/cloud-provider-openstack/pull/1600/commits/676bed90f314c5823a91ee7dd45ecf60c16851ff